content/en/platform/corda/5.2/deploying-operating/keys/_index.md

```diff
@@ -0,0 +1,198 @@
+    {\"configSecret\":{\"encryptedSecret\":\"cvNVJ4D7fNGm59Uh3Tlhi0W3pToDxl7vykUdZaxEuEPGUMdmu7G+xTEfl0CcC2R9aWl6FgrzmX5x+qCGNElUi/PBA40EGjEU\"}}}, \
+    {\"alias\":\"master2\",\"passphrase\":{\"configSecret\": {\"encryptedSecret\":\"gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag\"}},\"salt\": \
+    {\"configSecret\":{\"encryptedSecret\":\"flvAnf5G/aRmDIaahXHjftEpc0odufhzZsuiipxtQTFV+pccB+RHnIqle26jjDlDUBfn3HPbVezKhpRHL/NP5M+cX/rU15DE\"}}}]}, \
+    \"retrying\":{\"maxAttempts\": {\"default\": 3},\"waitBetweenMills\":{\"default\":[200]}}}}","schemaVersion": {"major":1, "minor":0}}' \
+    "$REST_API_URL/config"
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+
+   ```shell
+  Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f ${REST_API_USER}:${REST_API_PASSWORD})} -Method Put -Uri "$REST_API_URL/config" -Body (ConvertTo-Json -Depth 4 @{
+   {
+     "config": "{
+        "caching":{
+          "expireAfterAccessMins":{
+            "default":60
+          },
+          "maximumSize":{
+            "default":10000
+          }
+        },
+        "hsm":{
+          "defaultWrappingKey": "master1",
+          "retrying":{
+            "attemptTimeoutMills": 20000,
+            "maxAttempts": 3
+          },
+          "wrappingKeys":[{
+            "alias": "master1",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag"
+              }
+            },
+            "salt":{
+              "configSecret":
+              {
+                "encryptedSecret": "cvNVJ4D7fNGm59Uh3Tlhi0W3pToDxl7vykUdZaxEuEPGUMdmu7G+xTEfl0CcC2R9aWl6FgrzmX5x+qCGNElUi/PBA40EGjEU"
+              }
+            },
+          },
+          {
+            "alias": "master2",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "ZAmGcppwqUNsViSE06b801DVARbD9wT7SZnYP03WKZFOnC9KTsII04pWWq5Je2CUZdP+xIsHFUO3RkZVwhwnyQhWoxcW6JEo"
+              }
+            },
+            "salt":{
+              "configSecret":
+              {
+                "encryptedSecret": "flvAnf5G/aRmDIaahXHjftEpc0odufhzZsuiipxtQTFV+pccB+RHnIqle26jjDlDUBfn3HPbVezKhpRHL/NP5M+cX/rU15DE"
+              }
+            }
+          }]
+        "retrying":{
+          "maxAttempts":{
+            "default":3
+          },
+          "waitBetweenMills":{
+            "default":[200]
+          }
+        }
+      }",
+      "schemaVersion": {
+        "major": 1,
+        "minor": 0
+      },
+      "section": "corda.crypto",
+      "version": 1
+   })
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+   Corda wraps all new managed wrapping keys with this new master wrapping key.
```

Comment:
Minor: We have the following statement after step 3 `Corda rewraps all existing managed wrapping keys with the new master wrapping key.`

Shall we skip the statement in line 130 as it seems like a repetition?

Reply:
My understanding is that after step 2, all **new** managed wrapping keys will be rewrapped with the new master key. After step 3,  all **existing** managed wrapping keys are wrapped with the new master wrapping key.

I've modified it slightly to make this clearer.

Reply:
Second example added in https://github.com/corda/corda-docs-portal/pull/2047/commits/42466aad242375a9636bdeeac210ada7f6cbce91